app2/src/main/java/com/example/app2/pojo/SummaryItems.java

```diff
@@ -0,0 +1,16 @@
+package com.example.app2.pojo;
+
+import lombok.Data;
+
+@Data
+public class SummaryItems implements Cloneable {
```

Comment:
SummaryItems and SummaryLiquids are identical classes. Won't it be better to use one of them and adjust field names some how to increase readability when those properties are used in OrderProcessingService logic?